### ISSUE TITLE
Bug Report: weird cursor behaviour in Far Manager command line

### ISSUE THREAD

**Issue by luzhkovvv** [microsoft/terminal#2561]
# Environment

```
Windows build number: Microsoft Windows [Version 10.0.18362.295]
Windows Terminal version: 0.4.2382.0
Far Manager - practically any available, last tested on Far Manager v3.0 build 5454 x64 (2019-08-23) 
```

# Steps to reproduce

- install Far Manager from https://farmanager.com/download.php?l=en
- run Far Manager inside Windows Terminal (from cmd session or by creating new profile)
- type any text, look how cursor moves in command line while typing
- press Ctrl-O, try to move cursor using directions keys
- press "Enter" with any text entered in command line (no matter when in panels mode or in history mode, switched by Ctrl-O, no matter if correct command or any text)
- type any text again, look how cursor moves while typing
- press Ctrl-O, try to move cursor using directions keys

# Expected behavior

- cursor follows entered text
- cursor moves when "left" or "right" button pressed
- just like it is after fresh start

# Actual behavior

- cursor is displayed before last printed character, not after
- cursor does not moves when left or right buttons are pressed (just not displayed, internal position gets updated)

P.S. As I get it, it has something to do with the way Far clears and redraw the screen or with the way commands are executed in Far, but I have no ways to localize that bug in Windows Terminal or in Far. At least same version of Far works OK in plain cmd.exe or ConEmu.

**Comment by luzhkovvv** [comment]
Oops, it seems to be fixed by "Use Virtual Terminal for rendering" Far Manager option, just introduced in the latest version. Just for me that's enough, don't know if that "bug" still worth investigation.

**Comment by zadjii-msft** [comment]
Note that this _isn't_ fixed by #4372, and definitely still repros in 0.8 (and I suppose master (e8658cd) + #4372 as well)

**Comment by oleksabor** [comment]
I've just got the invisible cursor issue for Far manager. 
I've installed the latest Windows 10 19009 build and can't see cursor in the Far window. I had used Windows 10 1803 before and there was no problem with cursor.

The problem is that it looks like cursor is shown however it has black colour.

I had to change Far shortcut `Cursor colors` option on the `Terminal` tab. I set `Cursor color` to `inverse` and white cursor started to blink as before.
